src/_pytest/config/__init__.py

```diff
@@ -511,9 +516,10 @@ def _importconftest(self, conftestpath):
         # Using Path().resolve() is better than py.path.realpath because
         # it resolves to the correct path/drive in case-insensitive file systems (#5792)
         key = Path(str(conftestpath)).resolve()
-        try:
-            return self._conftestpath2mod[key]
-        except KeyError:
```

Comment:
iirc we could still keep this structure but instead do

```python
with contextlib.suppress(KeyError):
    return ...
...
```

Reply:
Great suggestion, it makes the code easier to follow, thanks.

I've also refactored the check for non-top-level `pytest_plugins` declaration, I believe it makes the code even easier to follow then.